docs-v2/integrations/all/intuit.mdx

```diff
@@ -14,32 +14,97 @@ import PreBuiltUseCases from "/snippets/generated/intuit/PreBuiltUseCases.mdx"
+  </Step>
+  <Step title="Test the authorization flow">
+    1. Use the [OAuth Playground](https://developer.intuit.com/app/developer/playground) to test your OAuth implementation.
+    2. The OAuth Playground provides sample data and allows you to preview each step of the authorization flow.
+    3. This step is optional but recommended to ensure your OAuth configuration is working correctly.
+  </Step>
+  <Step title="Next">
+    Follow the [_Quickstart_](/getting-started/quickstart).
+  </Step>
+</Steps>
 
 <Tip>Need help getting started? Get help in the [community](https://nango.dev/slack).</Tip>
 
-<Note>Contribute improvements to the setup guide by [editing this page](https://github.com/nangohq/nango/tree/master/docs-v2/integrations/all/intuit.mdx)</Note>
-
 
 ## Useful links
 
--   [How to register an Application](https://developer.intuit.com/app/developer/qbo/docs/develop/authentication-and-authorization/oauth-2.0#create-an-app)
--   [OAuth-related docs](https://developer.intuit.com/app/developer/qbo/docs/develop/authentication-and-authorization/oauth-2.0)
--   [List of OAuth scopes](https://developer.intuit.com/app/developer/qbo/docs/learn/scopes)
+-   [Intuit Developer Portal](https://developer.intuit.com/app/developer/homepage)
+-   [Intuit Developer Dashboard](https://developer.intuit.com/dashboard)
+-   [OAuth 2.0 Documentation](https://developer.intuit.com/app/developer/qbo/docs/develop/authentication-and-authorization/oauth-2.0)
+-   [OAuth Playground](https://developer.intuit.com/app/developer/playground)
+-   [OAuth Scopes](https://developer.intuit.com/app/developer/qbo/docs/learn/scopes)
+-   [Authorization Discovery Documents](https://developer.intuit.com/app/developer/qbo/docs/develop/authentication-and-authorization/discovery-document)
+-   [QuickBooks API Reference](https://developer.intuit.com/app/developer/qbo/docs/api/accounting/most-commonly-used/account)
+-   [API Explorer](https://developer.intuit.com/app/developer/qbo/docs/api/accounting/all-entities/account)
+-   [API Rate Limits](https://developer.intuit.com/app/developer/qbo/docs/develop/troubleshooting/error-handling#rate-limiting)
 
 <Note>Contribute useful links by [editing this page](https://github.com/nangohq/nango/tree/master/docs-v2/integrations/all/intuit.mdx)</Note>
 
-## API gotchas
+## Common Scopes
 
--   Refresh tokens have a rolling expiry of 100 days. If 100 days pass, or your refresh token expires, users need to go through the authorization flow again and reauthorize your app.
+| Scope | Description |
+| ----- | ----------- |
+| com.intuit.quickbooks.accounting | Access to QuickBooks Online accounting data |
+| com.intuit.quickbooks.payment | Access to QuickBooks Payments features |
+| openid | Required for OpenID Connect flows |
+| profile | Access to user profile information |
+| email | Access to user email information |
+| phone | Access to user phone information |
+| address | Access to user address information |
```

Comment:
Rather than listing the scopes, include a link to the list of scopes: https://developer.intuit.com/app/developer/qbo/docs/learn/scopes